pkg/controller/bundlec/controller.go

```diff
@@ -125,3 +194,77 @@ func (c *controllerIndexAdapter) ControllerByObject(gk schema.GroupKind, namespa
 	}
 	return objs, nil
 }
+
+func byConfigMapNamespaceNameIndex(obj interface{}) ([]string, error) {
+	d := obj.(*apps_v1.Deployment)
+	index := configMapNamespaceNameIndexKeysForContainers(d.Namespace, d.Spec.Template.Spec.Containers)
+	index = append(index, configMapNamespaceNameIndexKeysForContainers(d.Namespace, d.Spec.Template.Spec.InitContainers)...)
+	return index, nil
+}
+
+func configMapNamespaceNameIndexKeysForContainers(namespace string, containers []core_v1.Container) []string {
+	var indexKeys []string
+	for _, container := range containers {
+		for _, envFrom := range container.EnvFrom {
+			configMapRef := envFrom.ConfigMapRef
+			if configMapRef == nil {
+				continue
+			}
+			indexKeys = append(indexKeys, bySecretNamespaceNameIndexKey(namespace, configMapRef.Name))
```

Comment:
this should be `byConfigMapNamespaceNameIndexKey`, right?

same with the statement below.

Reply:
Doh, I checked it like 3 times and still got it wrong =) Thanks! Copy-pasting is bad.